assets/src/blocks/ActionsList/index.js

```diff
@@ -30,8 +30,7 @@ export const registerActionsListBlock = () => {
         pages: 0,
         perPage: 3,
         offset: 0,
-        order: 'desc',
```

Comment:
I've removed this filters since by default `orderby` is set to `date` and `order` is set to `desc`.

Check it out the [WP's official doc about arguments section](https://developer.wordpress.org/rest-api/reference/posts/#arguments)

Reply:
Also this filters are being overriden [here](https://github.com/greenpeace/planet4-master-theme/pull/2492/files#diff-5731321646bfd7fe6c8f5582c07ece1a8c4078579f40d518eafc522c4f89f421R66-R70)

Reply:
Last but not least, unfortunately passing an Array to `order` or `orderBy` within JS is not possible for now.
[These are a couple open issues](https://github.com/WordPress/gutenberg/issues/40170) that explains the need to set an array of strings, the same as we do in [PHP](https://github.com/greenpeace/planet4-master-theme/pull/2492/files#diff-5731321646bfd7fe6c8f5582c07ece1a8c4078579f40d518eafc522c4f89f421R66-R70).